packages/suite-web/e2e/tests/wallet/discovery.test.ts

```diff
@@ -29,6 +31,18 @@ describe('Discovery', () => {
         cy.log('all available networks should return something from discovery');
 
         cy.getTestElement('@dashboard/loading', { timeout: 1000 * 10 });
+
+        // wait randomly between 1000 and 4000 ms
+        cy.wait(getRandomInt(1, 40) * 100);
```

Comment:
This isn't 1000 to 4000 ms.
```suggestion
        // wait randomly between 1000 and 4000 ms
        cy.wait(getRandomInt(1, 4) * 1000);
```